MPF.Core/Data/Drive.cs

```diff
@@ -265,13 +265,13 @@ public static List<Drive> CreateListOfDrives(bool ignoreFixedDrives)
             }
 
             // Handle optical media by size and filesystem
-            if (this.TotalSize >= 0 && this.TotalSize < 800_000_000 && this.DriveFormat == "CDFS")
+            if (this.TotalSize >= 0 && this.TotalSize <= 800_000_000 && this.DriveFormat == "CDFS")
                 return (MediaType.CDROM, null);
-            else if (this.TotalSize >= 0 && this.TotalSize < 400_000_000 && this.DriveFormat == "UDF")
+            else if (this.TotalSize >= 0 && this.TotalSize <= 800_000_000 && this.DriveFormat == "UDF")
                 return (MediaType.CDROM, null);
-            else if (this.TotalSize >= 800_000_000 && this.TotalSize <= 8_540_000_000 && this.DriveFormat == "CDFS")
+            else if (this.TotalSize > 800_000_000 && this.TotalSize <= 8_540_000_000 && this.DriveFormat == "CDFS")
                 return (MediaType.DVD, null);
-            else if (this.TotalSize >= 400_000_000 && this.TotalSize <= 8_540_000_000 && this.DriveFormat == "UDF")
+            else if (this.TotalSize > 800_000_000 && this.TotalSize <= 8_540_000_000 && this.DriveFormat == "UDF")
                 return (MediaType.DVD, null);
```

Comment:
All of these checks can be condensed back down to 2 cases since the `this.DriveFormat` checks have been made irrelevant.

Reply:
Honestly, those are the only two "filesystems" that I've ever seen pop up during testing, so I'm pretty sure that the drive format doesn't get much more granular... unfortunately.